nautobot/project-static/css/base.css

```diff
@@ -934,7 +934,7 @@ code.hljs {
 .tiles {
     display: grid;
     grid-gap: 20px;
-    grid-template-columns: repeat(auto-fit, minmax(445px, 1fr));
+    grid-template-columns: repeat(auto-fill, minmax(445px, 1fr));
 }
```

Comment:
I missed that before when I worked on pages with many elements but `fill` feels more reasonable for tiles rather than `fit`. The former respects the number of grid columns regardless of number of elements inside, while the latter simply spans elements across the whole container width if there are more columns than chlidren. Here's comparison:

| `auto-fill` | `auto-fit` |
| - | - |
| ![fill](https://github.com/user-attachments/assets/92145f94-5afc-4b03-a984-61f389dcc0cd) | ![fit](https://github.com/user-attachments/assets/c186c3cc-8731-435b-aa97-2803bc2eea6e) |